.github/workflows/code_review.yml

```
@@ -0,0 +1,48 @@
+  code_review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Clone cody-code-reviewer repository
+        run: |
+          git clone https://github.com/codylabs/cody-code-reviewer.git
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)